### PR TITLE
Fix updates channel config

### PR DESCRIPTION
**Pull request by vkorn**

### CODE REVIEW

File: ironforgedbot/config.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def __init__(self):
         self.BOT_TOKEN: str = os.getenv("BOT_TOKEN", "")
         self.WOM_GROUP_ID: int = int(os.getenv("WOM_GROUP_ID") or 0)
         self.WOM_API_KEY: str = os.getenv("WOM_API_KEY", "")
-        self.RANKS_UPDATE_CHANNEL: int = int(os.getenv("RANKS_UPDATE_CHANNEL") or 0)
+        self.RANKS_UPDATE_CHANNEL: str = os.getenv("RANKS_UPDATE_CHANNEL")
```
Comment (incinn) [inline ironforgedbot/config.py:20]: My mistake.

Question: Would it not be better to use the channel id instead of the name? Would make it less brittle

**Review by incinn — COMMENTED** [review]
Can you address the formatting issues in the three files the lint workflow flagged.

If you haven't already, set it up in your editor to make life easier. Or, run in cli `black --check` then `black .`

File: ironforgedbot/config.py (hunk 1 above)
Comment (vkorn) [reply ironforgedbot/config.py:20]: Pure convenience -- easier to see channel name than id

**Comment by vkorn** [comment]
> Can you address the formatting issues in the three files the lint workflow flagged.
> 
> If you haven't already, set it up in your editor to make life easier. Or, run in cli `black --check` then `black .`
> 
> https://black.readthedocs.io/en/stable/integrations/editors.html

As I've mentioned before I don't like black and not going to use it locally. Feel free to pull these change in separately

**Pull request closed without merging**